### PR TITLE
Rename array shape constants

### PR DESCRIPTION
**Pull request by ehrenjulzert**
Resolves https://github.com/eclipse-openj9/openj9/issues/14280
Signed-off-by: Ehren Julien-Neitzert <ehren.julien-neitzert@ibm.com>

### CODE REVIEW

**Comment by ehrenjulzert** [comment]
@tajila

File: debugtools/DDR_VM/src/com/ibm/j9ddr/CompatibilityConstants29.dat [hunk 1]
```diff
@@ -76,3 +76,8 @@ MM_ForwardedHeader.OMR_FORWARDED_TAG_MASK = 0
 ShcdatatypesConstants.TYPE_PREREQ_CACHE = -1
 
 ShCFlags.J9SHR_DATA_TYPE_STARTUP_HINTS = -1
+
+J9ArrayShapeFlags.J9ArrayShape8Bit = 0
+J9ArrayShapeFlags.J9ArrayShape16Bit = 1
+J9ArrayShapeFlags.J9ArrayShape32Bit = 2
+J9ArrayShapeFlags.J9ArrayShape64Bit = 3
```
Comment (keithc-ca) [inline debugtools/DDR_VM/src/com/ibm/j9ddr/CompatibilityConstants29.dat:83]: I don't see any value in adding these constants here. We can wait until they're actually needed by DDR.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: debugtools/DDR_VM/src/com/ibm/j9ddr/CompatibilityConstants29.dat (hunk 1 above)
Comment (ehrenjulzert) [reply debugtools/DDR_VM/src/com/ibm/j9ddr/CompatibilityConstants29.dat]: Sounds good, fixed in f125181

**Comment by keithc-ca** [comment]
jenkins test sanity alinux64 jdk17

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**